### DISCUSSION TITLE
Zuverlässige SD-Card gesucht

### DISCUSSION THREAD

**Discussion by kalwados** [jomjol/AI-on-the-edge-device#2595] [General]
Zum zweitem Mal innerhalb eines Jahres ist eine 4GB-Micro-SD-Karte im ESP32-CAM gestorben. OK, war ein Billigprodukt von "Claudisk"; gabs mal im Doppelback bei Am*z*n. Warum sterben die Teile eigentlich wie die Fliegen; und welche zuverlässigeren Alternativen gibt es denn?  Laut Doku soll ESP32-CAM ja max. 16 GB unterstützen; da ist die Auswahl nicht besonders groß...
Welche SD-Card könnt ihr denn so empfehlen?

TIA
Kalwados

**Comment by Zenyte-zz** [comment]
Bei mir läuft eine 32GB Sandisk jetzt (3 Tage) ohne Probleme. Also scheinen auch größere Karten zu funktionieren.

**Comment by nhet** [comment]
Ich habe ähnliche Probleme mit meinen SD-Karten. Zu der Größe habe ich Folgendes gefunden:
https://www.esp32.com/viewtopic.php?t=14492#p58373
Vielleicht kann man da was einbauen und testen?

**Comment by Jamaka27** [comment]
Hallo, Nach dem ich das selbe Problem hatte, stellte ich meinen Router direkt zum esp, also ohne Repeater, da nach ging es ohne Probleme. Vorher konnte ich teilweise nicht mal das startsetup ausführen und wenn dann nur sehr sehr zäh. Eventuell hilft dies bei der Einrichtung. Da nach ging es auch mit Repeater. Cheers, mladiVon meinem HONOR-Telefon gesendet-------- Ursprüngliche Nachricht --------Von: nhet ***@***.***>Datum: Di., 19. Sep. 2023, 09:32An: jomjol/AI-on-the-edge-device ***@***.***>Cc: Subscribed ***@***.***>Betreff: Re: [jomjol/AI-on-the-edge-device] Zuverlässige SD-Card gesucht (Discussion #2595)
Ich habe ähnliche Probleme mit meinen SD-Karten. Zu der Größe habe ich Folgendes gefunden:
https://www.esp32.com/viewtopic.php?t=14492#p58373
Vielleicht kann man da was einbauen und testen?
